TaskModules/powershell/TaskModuleSqlUtility/SqlQueryOnTargetMachines.ps1

```diff
@@ -106,7 +68,10 @@ function Execute-SqlQueryDeployment
             }
         }
 
-        Invoke-Sqlcmd @spaltArguments
+        $additionalArguments = $additionalArguments.Replace("`$", "```$")
```

Comment:
Can you add some L0 for this? Can you ensure we have validated this with additional arguments having special characters.